### PR TITLE
slience warnings of sklearn

### PR DESCRIPTION
**Pull request by QianqianHan96**
#16  
I slienced the warnings of sklearn in notebook https://github.com/EcoExtreML/Emulator/blob/slience-warnings-sklearn/2daskParallel/1002_1year_10degree.ipynb. 
I also changed the rf model path inside map_block function in this notebook.
@geek-yang Yang, could you help me review this PR? Thanks!

### CODE REVIEW

File: 2daskParallel/.ipynb_checkpoints/1002_1year_10degree-checkpoint.ipynb
Comment (geek-yang) [inline 2daskParallel/.ipynb_checkpoints/1002_1year_10degree-checkpoint.ipynb:1]: This is a caching file. To make the repo clean, it is better to exclude these files. Please remove the `.ipynb_checkpoints` folder in this PR.

Also for the [`.ipynb_checkpoints`](https://github.com/EcoExtreML/Emulator/tree/main/1sbatchParallel/.ipynb_checkpoints) folder in`1sbatchParallel`, just move it too.

Btw, if you add `.ipynb_checkpoints` in the `.gitignore`, then next time when you make commits these caching files won't be counted by git, which makes life easier 😉 .

**Review by geek-yang — CHANGES_REQUESTED** [review]
Looks good to me. Just a minor comment about the caching file. Please remove it from the PR, to keep the repo clean.

I didn't run the notebook myself but I checked your results. The warnings are gone. But I notice that you didn't add feature name as a row to the pandas dataframe. Just wonder what did you actually do to silent the warnings? Did you remove the feature names in the pickle model?

File: 2daskParallel/.ipynb_checkpoints/1002_1year_10degree-checkpoint.ipynb
Comment (QianqianHan96) [reply 2daskParallel/.ipynb_checkpoints/1002_1year_10degree-checkpoint.ipynb:1]: Thanks for your careful check, Yang. I have deleted all `.ipynb_checkpoints` folders in this PR and modified `.gitignore` as well. Please review the changes, Thanks!

**Review by geek-yang — APPROVED** [review]
Looks cool! Thanks for addressing my comments! Let's merge it!

**Comment by geek-yang** [comment]
> > Looks good to me. Just a minor comment about the caching file. Please remove it from the PR, to keep the repo clean.
> > I didn't run the notebook myself but I checked your results. The warnings are gone. But I notice that you didn't add feature name as a row to the pandas dataframe. Just wonder what did you actually do to silent the warnings? Did you remove the feature names in the pickle model?
> 
> Hi Yang, I added feature names as a row in the following part (first screenshot is the old version and second screenshot is the new version), maybe you did not notice it. I did not remove feature names in the pickle model. Please let me know is this okay. Thanks!
> 
> image image

Indeed, I missed this line. Nice work!

**Pull request merged**